core/src/main/java/com/sequenceiq/cloudbreak/service/image/CustomImageCatalogService.java

```diff
@@ -61,6 +64,18 @@ public CustomImage getCustomImage(Long workspaceId, String imageCatalogName, Str
         return getCustomImageFromCatalog(imageCatalog, imageId);
     }
 
+    public Image getSourceImage(CustomImage image, String catalogName) {
+        String imageId = image.getCustomizedImageId();
+        LOGGER.debug(String.format("Get source image '%s' from default catalog", imageId));
+        try {
+            StatedImage sourceStatedImage = imageCatalogService.getSourceImageByImageType(image, catalogName);
```

Comment:
catalogName is the catalog name of the custom image. source image should come from the default image catalog

Reply:
Nice catch, I didn't notice it!